src/plugins/expression_reveal_image/public/components/reveal_image_component.tsx

```diff
@@ -60,12 +63,15 @@ function RevealImageComponent({
   }, [imgRef, handlers]);
 
   useEffect(() => {
```

Comment:
this effect looks obsolete now, handlers will never change

Reply:
That code is for defining `componentDidMount` lifecycle.

Reply:
I removed it, because `done` is already calling after updating the dimensions of parentNode.